BUILD.bazel

```diff
@@ -86,8 +86,8 @@ copy_to(
         ],
     }),
     rename = {
-        "armeabi-v7a.apk": "gapid-armeabi.apk",
-        "arm64-v8a.apk": "gapid-aarch64.apk",
+        "armeabi-v7a.apk": "gapid-armeabi-v7a.apk",
```

Comment:
We might have to update the packaging scripts in the kokoro directory (I am not actually sure about this).